adaptdl/adaptdl/torch/adascale.py

```diff
@@ -217,9 +218,10 @@ def _backward_hook(self, idx, grad):
             self._local_sqr = torch.zeros(
                 (self._accumulation_steps, len(self._optimizer.param_groups)),
                 device=grad.device)
+        if self._mp_scaler is not None:
+            grad = grad / self._mp_scaler.get_scale()
```

Comment:
Last I checked, modifying the gradient here like this doesn't actually work..

Reply:
This is purely for the values stored in `self._local_sqr`. It's a rebinding of `grad` rather than an update.